assessor/views.py

```diff
@@ -0,0 +1,15 @@
+from django.views.decorators.http import require_GET
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.decorators import api_view, permission_classes
+
+
+@api_view(['GET'])
+@require_GET
+@permission_classes([IsAuthenticated])
+def assessor_dashboard(request):
+
```

Comment:
Hi, Indi! Following my comment on Arya's pull request, I would also like to suggest that you first make sure that the accessing user is an Assessor. The permission_classes([IsAuthenticated]) only make sure that a valid Authorization header is present in the request, thus the authorization for ensuring that the request.user is an Assessor must be done manually.

Reply:
Thanks for your suggestion, Jo! I'll try to implement custom permissions.